app/js/home/components/Hero.jsx

```diff
@@ -33,8 +33,8 @@ class Hero extends React.Component {
           <div className="fancy-hero-container" />
           <div className="hero-content">
             <h3>Weekly Meetings</h3>
-            <h6>Mondays @ 4:30pm</h6>
-            <h6>GOL-1670</h6>
+            <h6>Thursdays @ 4:00pm</h6>
+            <h6><a href={"sse.rit.edu/go/generalmeeting"}>Click here to join the Zoom</a></h6>
```

Comment:
Why the curly brackets?
```suggestion
            <h6><a href="sse.rit.edu/go/generalmeeting">Click here to join the Zoom</a></h6>
```

Reply:
You may want to consider relative paths rather than absolute ones for this and the new link in `Page.jsx`

If possible, it would be nice to have them be react router links: https://reactrouter.com/web/api/Link

but it's totally fine as is.

Reply:
curly bois were just habit I can get rid of. In terms of using react router I dont think we should worry about that. At least for these two links they take you away from the sse website. One to zoom and one to google slides.